### PR TITLE
[Pages] Update web application user interface

### PR DESCRIPTION
**Pull request by hankpeeples**

### CODE REVIEW

File: Application/src/Components/MyPoster.css [hunk 1]
```diff
@@ -111,4 +108,4 @@
     display: flex;
     justify-content: center;
     margin: 1em;
-}
\ No newline at end of file
+}
```
Comment (hankpeeples) [inline Application/src/Components/MyPoster.css:111]: CleanShot 2022-07-18 at 16 47 09@2x

File: Application/src/Components/BarGraph.js [hunk 2]
```diff
@@ -1,43 +1,41 @@
...
+          },
+        }}
+      />
+    </>
+  );
 }
 
-export default BarGraph;
\ No newline at end of file
+export default BarGraph;
```
Comment (hankpeeples) [inline Application/src/Components/BarGraph.js:41]: CleanShot 2022-07-18 at 23 18 14@2x

**Comment by TylerAdamMartinez** [comment]
Can you make all the images stay the same size on the poster card and zoomed in with the same aspect ratio in the pop-up version of the poster? In addition, please add a material icon trash can icon or equivalent for the delete function and a material icon gear or tool icon for the edit function inside the poster pop-up.

File: Application/src/Components/MyPoster.css (hunk 1 above)
Comment (TylerAdamMartinez) [reply Application/src/Components/MyPoster.css:111]: Does the poster card change depending on the tallest poster giving or have you fixed that bug?

Comment (hankpeeples) [reply Application/src/Components/MyPoster.css:111]: Every image will be fixed to that size at the moment.

Comment (TylerAdamMartinez) [reply Application/src/Components/MyPoster.css:111]: To the square aspect ratio?

Comment (hankpeeples) [reply Application/src/Components/MyPoster.css:111]: Yeah, I don't know CSS all that well because I never use it.

Comment (TylerAdamMartinez) [reply Application/src/Components/MyPoster.css:111]: You're good

File: Application/src/Components/MyPoster.js [hunk 3]
```diff
@@ -6,47 +6,62 @@ import { useState, useRef } from 'react';
...
+          </div>
+        </div>
+      </div>
+    </Popup>
+  );
 }
 
-export default MyPoster;
\ No newline at end of file
+export default MyPoster;
```
Comment (hankpeeples) [inline Application/src/Components/MyPoster.js:67]: CleanShot 2022-07-19 at 12 08 44@2x

CleanShot 2022-07-19 at 12 09 24@2x

**Comment by TylerAdamMartinez** [comment]
Would mind if I made a change in this branch that says "loading image" on image change and "generating report" on submitting handler back to submit after it's been sent is completed?

File: Application/src/Components/MyPoster.js [hunk 4]
```diff
@@ -52,6 +54,18 @@ function MyPoster(props) {
...
+              className='deleteIcon'
+              fontSize='large'
+              onClick={() => console.log('Delete pressed')}
+            />
+            <EditIcon
+              className='editIcon'
+              fontSize='large'
+              onClick={() => console.log('Edit pressed')}
+            />
+          </div>
```
Comment (hankpeeples) [inline Application/src/Components/MyPoster.js:68]: CleanShot 2022-07-19 at 12 24 39@2x

I don't really like the position of the icons but that's the best I could come up with. Feel free to move them if you want.

**Comment by hankpeeples** [comment]
> Would mind if I made a change in this branch that says "loading image" on image change and "generating report" on submitting handler back to submit after it's been sent is completed?

Github won't let me reply directly but yeah that's fine with me.

**Comment by TylerAdamMartinez** [comment]
https://user-images.githubusercontent.com/57375362/179849289-5b297f29-c666-46cf-94aa-3f65b22f096e.mov

**Review by TylerAdamMartinez — APPROVED** [review]
Incredible work here!!

**Pull request merged**